core/data/src/main/java/com/example/poc/core/data/credentials/CredentialsRemoteDataSource.kt

```diff
@@ -2,11 +2,27 @@ package com.example.poc.core.data.credentials
 
 interface CredentialsRemoteDataSource {
 
+    /**
+     * Retrieves an user from the RockSpoon merchant web service
+     * @param request GetCredentialsRequest.Email or GetCredentialsRequest.PinCode
+     */
+    suspend fun getCredentials(request: GetCredentialsRequest): Credentials?
```

Comment:
why do you need this method if you already has getCredentials and getCredentialsByPinCode?

Reply:
just keep this method or the other 2